### ISSUE TITLE
[Bulk Order Management] Orders made after Order Cycle closed not showing 

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/openfoodnetwork#10082]
## Description
When you create orders for clients after the end of an Order Cycle (OC), and you link these orders to the closed Order Cycle, these orders are not showing in the Bulk Order Management (BOM) page when clicking on this Order Cycle in the dropdown list. 
They are only displayed when you select the dates. 

This has been raised by a shop manager who often pass orders for his clients who forgot to do it while the OC was ongoing. 

## Expected Behavior
Orders linked to an Order Cycle should be displayed in the Bulk Order Management page, whether you choose dates or the specific Order Cycle. 

## Actual Behaviour
On the BOM page, orders linked to an OC but that has been done from dashboard by the shop manager after the end of the OC don't show when you select this OC in the dropdown list.

## Steps to Reproduce
1. Create a shop
2. Open an OC 
3. Pass an order while the OC is ongoing
4. Close the OC (change end date) 
5. Pass an order from dashboard (/admin/orders/new), select the shop as distributor and select the specific OC
6. Go to the BOM page, select a date range including the date of your orders, see that the orders made after OC was closed are displayed
7. Now select the specific OC, see that the orders made after OC was closed aren't displayed. 
There's another bug: if you made orders previously on this OC, they will show even if they are out of the date range. I'll create another issue for this.

### step 6
![Screen Shot 2022-12-01 at 19 48 09](https://user-images.githubusercontent.com/28386648/205137710-0a41c68e-f301-4758-8530-8de661d8b960.png)

### step 7
![Screen Shot 2022-12-01 at 19 48 25](https://user-images.githubusercontent.com/28386648/205137716-c5d53fd8-db5f-4572-86fd-0705fd5d8dd5.png)

## Workaround
Don't select the OC

## Severity
bug-s3: a feature is broken but there is a workaround

**Comment by RachL** [comment]
@audez are you sure there isn't any other step you have made? I've run twice the same test and the orders are showing. I can't reproduce at this stage.

**Comment by audez** [comment]
yes. What is the date you chose when changing the OC end date? It seems to come from there. I didn't reproduce only if I chose an end date which is the same date (but earlier time) than the "after closing" orders.  

It's tricky to find the exact case and the issue origin though because of the 2 other bugs:
#10091 
#10092

**Comment by audez** [comment]
For more context this is the issue raised by the user, on FR prod:
OFN ID: 2931
Sales cycle: Friday 16 December - *** (from 03/11/2022 to
27/11/2022)
Three orders were added to the sales cycle on 11/30/2022, they don't appear on BOM page when selecting the OC

**Comment by RachL** [comment]
@audez do you have the order numbers?

**Comment by audez** [comment]
yep R313872714, R638141701, R183503570
